GitCommands/PathUtil.cs

```diff
@@ -232,14 +232,7 @@ public static string GetDisplayPath([NotNull] string path)
             // TODO verify whether the user profile contains forwards/backwards slashes on other platforms
             var userProfile = Environment.GetFolderPath(Environment.SpecialFolder.UserProfile);
 
-            var comparison = EnvUtils.RunningOnWindows() ? StringComparison.OrdinalIgnoreCase : StringComparison.Ordinal;
-
-            if (TryGetExactPath(path, out var exactPath))
-            {
-                path = exactPath;
-            }
-
-            if (path.StartsWith(userProfile, comparison))
+            if (path.StartsWith(userProfile, StringComparison.OrdinalIgnoreCase))
```

Comment:
Any reason we don't want to check for case sensitive/insensitive based on OS? Don't we want to support Linux/Mac for the future?

Reply:
Whilst not strictly necessary at this point, it would be good to keep x-plat in mind

Reply:
> Even for linux, I can hardly image that the userprofile path is there with different casings.

I suppose that's true. I don't have a strong opinion on this. I suppose it's fine then.